File: JavaScript/exercicio-09/solucao.js
const prompt = require("prompt-sync")();
const readline = require("readline-sync");

function converterMoeda() {
  const reaisDisponiveis = parseFloat(prompt('Digite o valor em reais que voce tem na sua carteira: '));
  const valorDolar = reaisDisponiveis / 3.45; 

  console.log(`Voce pode comprar ${valorDolar.toFixed(2)}USD`);
  readline.question('Digite Enter para continuar ... ');
}

converterMoeda();

File: Python/exercicio-09/solucao.py
def converter_moedas():
     reais_disponiveis = float(input('Digite o valor em reais que voce tem na sua carteira: '));
     valor_dolar = reais_disponiveis / 3.45; 

     print(f'Voce pode comprar {valor_dolar:.2f} USD')
     input("Digite Enter para sair ... ")


converter_moedas()
